source/views/calendar/event-detail.js

```diff
@@ -34,6 +36,34 @@ function MaybeSection({header, content}: {header: string, content: string}) {
     : null
 }
 
+function Links({header, content}: {header: string, content: string}) {
+  const rawUrls = getUrls(content)
+  const links = Array.from(rawUrls)
```

Comment:
It's returned in a `Set` so rather than write an iterator, I convert it to an array.